scopes/workspace/watcher/watch.cmd.ts

```diff
@@ -23,8 +23,9 @@ export class WatchCommand implements Command {
   name = 'watch';
   description = 'automatically recompile modified components (on save)';
   extendedDescription = `by default, the watcher doesn't use polling, to keep the CPU idle.
-in some rare cases, this could result in missing file events. to fix it, try to prefix this command with CHOKIDAR_USEPOLLING=true
-alternatively, restarting the computer could also help.`;
+in some rare cases, this could result in missing file events (files are not watched).
+to fix it, try to stop other watchers on the same machine.
+alternatively, to use polling, run "bit config watch_use_polling true".`;
```

Comment:
@davidfirst you forgot the word set in bit config set

Reply:
Thanks! :) fixed.